sass/_layout.scss

```diff
@@ -0,0 +1,76 @@
+.container {
+    background-color: $color-divider;
+    width: 100vw;
+    height: 900px; // later will be defined by the content
```

Comment:
Instead of fixed height you could use min-height: 100vh;
In this case even if the content is not tall enough, your container will always have the height of the screen